src/api.js

```diff
@@ -13,4 +39,9 @@ export function getRandomPerson(filter = 'all') {
       return responseJson;
     }
   })
+
+  // // Stub Code to Work on Front-End only by not hitting the API
+  // let data = people[Math.ceil(Math.random() * 100) % people.length]
+  // return new Promise((res, rej) => res(data));
+  // // Comment this out and un-comment actual implementation before commits
```

Comment:
I wonder if we can have an environment variable to control if this function calls the API or uses the placeholder data? I have no idea how to do that in javascript!